lib/storage/file.js

```diff
@@ -439,145 +441,129 @@ File.prototype.createReadStream = function(options) {
     md5 = options.validation === 'md5';
   }
 
-  if (is.undef(options.validation)) {
-    crc32c = true;
-    md5 = true;
+  if (rangeRequest) {
+    // Range requests can't receive data integrity checks.
```

Comment:
Do you think it would be useful to throw an exception here if the developer had set some form of validation? While that would obviously be a user error, it might be nice to let them know that what they're trying to do isn't possible.